modules/common-attributes.adoc

```diff
@@ -41,5 +41,4 @@ endif::[]
 :launch: image:app-launcher.png[title="Application Launcher"]
 :mtc-short: MTC
 :mtc-full: Migration Toolkit for Containers
-:mtc-version: 1.5
-:mtc-version-z: 1.5.1
```

Comment:
Is `mtc-version-z` used anywhere in the docs? If so, does it need to be scrubbed?

Reply:
It used to be relevant. Now it isn't.